assets/blocks/BouncyBlock.block

```diff
@@ -0,0 +1,19 @@
+{
+    "displayName": "Bouncy block",
+    "attachmentAllowed" : true,
+    "shadowCasting" : false,
+    "rotation" : "horizontal",
+    "categories" : ["BouncyBlock"],
+    "hardness": 15,
+    "tiles": {
+      "top": "funnyBlocks:BouncyBlock",
+      "sides": "funnyBlocks:BouncyBlock_side",
```

Comment:
Suggestion: Rename `BouncyBlock_side` to `BouncyBlockSide` for consistency (both the file and the entry here)